### PR TITLE
Add common uv_udp_* error checking.

### PR DESCRIPTION
**Pull request by txdv**
This makes uv_udp_\* error code checking independent of the platform.
Do not pull immediately, currently this is only the linux version.

### CODE REVIEW

File: src/uv-common.c [hunk 1]
```diff
@@ -264,6 +262,46 @@ int uv_tcp_connect6(uv_connect_t* req,
 }
 
 
+int uv_udp_send(uv_udp_send_t* req, uv_udp_t* handle,
+                uv_buf_t bufs[], int bufcnt, struct sockaddr_in addr,
+                uv_udp_send_cb send_cb) {
```
Comment (bnoordhuis) [inline src/uv-common.c]: Style: each argument on its own line.

Comment (txdv) [inline src/uv-common.c]: the methods below too?

Comment (bnoordhuis) [inline src/uv-common.c]: Yep.

**Comment by bnoordhuis** [comment]
Landed in c5101ae, thanks.

**Pull request closed without merging**